src/plugins/credential_management/public/components/edit_credential/edit_credential.tsx

```diff
@@ -20,13 +19,24 @@ import {
   EuiButton,
   EuiPageContent,
   EuiFieldPassword,
+  EuiFlexItem,
+  EuiToolTip,
+  EuiButtonIcon,
+  EuiConfirmModal,
 } from '@elastic/eui';
 import { DocLinksStart } from 'src/core/public';
 import { getCreateBreadcrumbs } from '../breadcrumbs';
 import { CredentialManagmentContextValue } from '../../types';
 // TODO: Add Header https://github.com/opensearch-project/OpenSearch-Dashboards/issues/2051
 import { context as contextType } from '../../../../opensearch_dashboards_react/public';
 import { CredentialEditPageItem } from '../types';
+import { deleteCredentialDescribeMsg } from '../text_content/text_content';
+import { deleteCredentialConfirmMsg } from '../text_content/text_content';
+import { deleteCredentialWarnMsg } from '../text_content/text_content';
+import { confirmTitle } from '../text_content/text_content';
+import { deleteCredentialButtonDescription } from '../text_content/text_content';
+import { cancelButtonText } from '../text_content/text_content';
+import { confirmButtonText } from '../text_content/text_content';
```

Comment:
let's follow some code convention for import https://google.github.io/styleguide/tsguide.html#imports